src/openzaak/components/documenten/tests/test_registered_document.py

```diff
@@ -0,0 +1,418 @@
+        self.assertEqual(eio.inhoud.read(), b"some file content")
+        self.assertEqual(eio.link, "http://een.link")
+        self.assertEqual(eio.beschrijving, "test_beschrijving")
+        self.assertEqual(eio.informatieobjecttype, self.informatieobjecttype)
+        self.assertEqual(eio.vertrouwelijkheidaanduiding, "openbaar")
+        self.assertEqual(eio.verschijningsvorm, "Vorm A")
+        self.assertEqual(eio.trefwoorden, ["some", "other"])
+
+        zio = ZaakInformatieObject.objects.get()
+        self.assertEqual(zio.zaak, self.zaak)
+        self.assertEqual(zio.aard_relatie, RelatieAarden.hoort_bij)
+        self.assertEqual(zio.status, self.status)
+        self.assertEqual(
+            zio.vernietigingsdatum.isoformat(), "2019-08-24T14:15:22+00:00"
+        )
+
+        expected_eio_url = reverse(eio)
+        expected_file_url = get_operation_url(
+            "enkelvoudiginformatieobject_download", uuid=eio.uuid
+        )
+
+        expected_zio_url = reverse(zio)
+
+        expected_response = {
+            "enkelvoudiginformatieobject": content["enkelvoudiginformatieobject"]
+            | {
+                "url": f"http://testserver{expected_eio_url}",
+                "inhoud": f"http://testserver{expected_file_url}?versie=1",
+                "versie": 1,
+                "bestandsdelen": [],
+                "beginRegistratie": eio.begin_registratie.isoformat().replace(
+                    "+00:00", "Z"
+                ),
+                "vertrouwelijkheidaanduiding": "openbaar",
+                "bestandsomvang": eio.inhoud.size,
+                "integriteit": {"algoritme": "", "waarde": "", "datum": None},
+                "ontvangstdatum": None,
+                "verzenddatum": None,
+                "ondertekening": {"soort": "", "datum": None},
+                "indicatieGebruiksrecht": None,
+                "status": "",
+                "locked": False,
+                "lock": "",
+                "verschijningsvorm": "Vorm A",
+            },
+            "zaakinformatieobject": content["zaakinformatieobject"]
+            | {
+                "url": f"http://testserver{expected_zio_url}",
+                "uuid": str(zio.uuid),
+                "informatieobject": f"http://testserver{expected_eio_url}",
+                "titel": "string",
+                "beschrijving": "string",
+                "registratiedatum": "2025-01-01T12:00:00Z",
+                "aardRelatieWeergave": RelatieAarden.hoort_bij.label,
+                "vernietigingsdatum": "2019-08-24T14:15:22Z",
+            },
+        }
+
+        response_data = response.json()
+        self.assertEqual(sorted(response_data.keys()), sorted(expected_response.keys()))
+
+        for obj in response_data:
+            for key in response_data[obj]:
+                with self.subTest(field=key):
+                    self.assertEqual(
+                        response_data[obj][key], expected_response[obj][key]
+                    )
```

Comment:
Can't you do `self.assertEqual(response_data, expected_response)`? If you set maxDiff = None on the class it should display errors clearly

Reply:
removed, originally did it this way for consistency with other tests